lib/daru/index.rb

```diff
@@ -343,7 +345,7 @@ def values
     end
 
     def inspect
-      "Daru::MultiIndex:#{object_id} (levels: #{levels}\nlabels: #{labels})"
+      "#<Daru::MultiIndex(#{levels.map { |l| l.map(&:inspect).join(',') }.join(' > ')})>"
```

Comment:
For the record, here is two for comparison:
```
#<Daru::MultiIndex(:a,:b,:c > :one,:two > :bar,:baz,:foo)>

#<Daru::MultiIndex:83987740 (levels: [[:a, :b, :c], [:one, :two], [:bar, :baz, :foo]]\nlabels: [[0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2], [0, 0, 1, 1, 0, 1, 1, 0, 0, 0, 1, 1], [0, 1, 0, 1, 0, 0, 1, 2, 0, 1, 2, 0]])>
```

I'm still think that first is much better. Though, I'll leave it in current PR, and provide another one, with just `inspect`s fixed, with detailed reasoning, and we could discuss it there or just throw away :)